pygeo/pyBlock.py

```diff
@@ -789,13 +789,16 @@ def attachPoints(self, coordinates, ptSetName, interiorOnly=False, faceFreeze=No
             kwargs pass through to the actual projectPoints() function
         """
 
+        # Unpack kwargs
+        nIter = kwargs.get("nIter", 100)
+
```

Comment:
Given that there are no additional kwargs, we could just pass `nIter` as explicit argument to `atttachPoints`

Reply:
This is related to the `comm` kwargs discussion above. `attachPoints` is called in `addPointSet`
https://github.com/mdolab/pygeo/blob/70ae801f137fe7c0be1d4a72214842e4c8ab0f29/pygeo/parameterization/DVGeo.py#L616-L619
We can make it can explicit argument in attachPoints, but then we simply move the unpacking up one level to `addPointSet`.

Reply:
I see, this is fine as is then